doc/release/technotes/README.extern.rst

```diff
@@ -37,26 +36,28 @@ All of the following sections rely on the same general approach:
    For example, if an external function foo() was defined in foo.h and
    foo.c, it could be added to the compilation using either:
 
+   ::
```

Comment:
These look fine as is, but you can also use `.. ::code-block sh` for syntax recognized shell commands.